javascript/packages/orchestrator/src/test-runner/assertions.ts

```diff
@@ -320,9 +324,11 @@ const CustomJs = ({
     } catch (err: any) {
       console.log(
         `\n ${decorators.red(
-          `Error running script: ${file_path!}`,
+          `Error importing/running script: ${file_path!}`,
```

Comment:
Probably can leave this line unchanged:

```suggestion
          `Error running script: ${file_path!}`,
```

Reply:
Yes, from user perspective is the same :)